<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>430</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Eco-Fridge</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../나뭇잎.jpg</normaloff>../나뭇잎.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>20</width>
     <height>430</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>카페24 고운밤</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>냉동고</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>카페24 고운밤</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>냉장고</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cafe24 Oneprettynight</family>
     <pointsize>10</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>slot_freezer()</slot>
  <slot>slot_refrigerator()</slot>
 </slots>
</ui>
